Meadowlark-js/packages/meadowlark-authz-server/src/handler/CreateClient.ts

```diff
@@ -119,7 +119,19 @@ export async function createClient(authorizationRequest: AuthorizationRequest):
       return { body: { error }, statusCode: 400 };
     }
 
-    const validation: BodyValidation = validateCreateClientBody(parsedBody);
+    let validation: BodyValidation = validateCreateClientBody(parsedBody);
+    if (!validation.isValid && validation.suggestions) {
+      writeDebugStatusToLog(
+        moduleName,
+        authorizationRequest,
+        'createClient',
+        400,
+        'Invalid request body, checking for suggestions',
+      );
+      parsedBody = applySuggestions(parsedBody, validation.suggestions);
+      validation = validateCreateClientBody(parsedBody);
+    }
```

Comment:
This new chunk of code would benefit from an explanation of why we are doing this.